README.md

```diff
@@ -7,18 +7,18 @@
 
 The instructions in the 'install dependencies' section should be run once to get the dependencies installed.  The next section, 'Build PolyMPO', details how to do an initial build of polyMPO.  After that, the instructions in the 'develop and rebuild' section should be followed to rebuild polyMPO after making source code changes.
 
-An NVIDIA GPU is required for building and running the software.  CUDA flags for NVIDIA GPUs with the Turing architecture are included below.  For scorec users Turing GPUs are in the `cranium` and `blockade` workstations.  More info on how to set these flags for different NVIDIA architectures is below.
+An NVIDIA GPU is required for building and running the software.  CUDA flags for NVIDIA GPUs with the Turing architecture are included below.  For SCOREC users, a Turing GPU is in the `checkers` workstation.  More info on how to set these flags for different NVIDIA architectures is below.
```

Comment:
```suggestion
An NVIDIA GPU is required for building and running the software.  CUDA flags for NVIDIA GPUs with the Turing architecture are included below.  For SCOREC users, Turing GPUs are in the `cranium` and `blockade` workstations.  More info on how to set these flags for different NVIDIA architectures is below.
```

It looks like our changes crossed paths.  For the record, checkers has an Ampere GPU with compute capability 8.6.

Reply:
That's interesting because my build still works with this flag `-DKokkos_ARCH_TURING75=ON`